### ISSUE TITLE
[Bug]: IFC won't load because there is no relatedIDs array ?

### ISSUE THREAD

**Issue by plgingras88** [ThatOpen/engine_web-ifc#1201]
### What happened?

I installed the very basic example of the IFC reader with pnpm create bim-app@latest. It worked all fine for my first IFC import attempt, then the second file would import but not load in the canvas, neither the sidebar tree. Once the import is completed, I get the following error in the console :

![image](https://github.com/user-attachments/assets/ba93d5ef-ee22-402d-8e25-e4a4b10cea05)
![image](https://github.com/user-attachments/assets/7081f0eb-41ba-40b3-9eba-ab614be2a340)
![image](https://github.com/user-attachments/assets/6640ab42-7525-4f36-a405-8c85ab8b2c35)

But if I change the errored line for const relatedIDs = []; like below, it loads fine in the canvas.

![image](https://github.com/user-attachments/assets/dd59cf9c-42af-49a7-a6da-71b084356770)

I will see if I can get the permission to provide the IFC file to help you debugging this issue.

### Version

0.0.66

### Relevant log output

```shell
See screenshots
```

**Comment by beachtom** [comment]
I don't think this is a bug in the low level IFC parsing - but in components. So I will transfer the issue there so the guys there can take a look
